dosagelib/plugins/x.py

```diff
@@ -28,8 +28,3 @@ def imageUrlModifier(self, url, data):
         if url and '/large/' in data:
             return url.replace(".png", "_large.png")
         return url
-
-    def shouldSkipUrl(self, url, data):
-        return url in (
-            self.stripUrl % '1663',  # Garden
-        )
```

Comment:
Interesting. There was once a time (before 2013, LOL), where failed image fetches were fatal - but nowadays we can probably remove all those methods and just keep the error message in the log file...

Reply:
I think there are some cases where it may be worth keeping, such as blogs that aren't always comic pages, but this wasn't one of them.